### PR TITLE
PCHR-571: Move the toil button to the left

### PR DESCRIPTION
**Pull request by danilo-compucorp**
pchr-571

### CODE REVIEW

**Comment by AkA84** [comment]
Please provide also  before/after screenshots for tablet and mobile viewports

File: civihr_employee_portal/src/Blocks/AbsenceLinks.php [hunk 1]
```diff
@@ -16,13 +16,14 @@ public function generateBlock() {
         $links = '';
         $links .= '<div class="chr_panel__footer">';
         $links .= '<div class="chr_actions-wrapper">';
+        $links .= civihr_employee_portal_make_link(t('Request TOIL'), 'credit', '', 'pull-left');
```
Comment (AkA84) [inline civihr_employee_portal/src/Blocks/AbsenceLinks.php]: You moved this outside of `.chr_actions-wrapper__inline-duo`, so probably the end result in smaller viewports has changed, please check that

**Comment by danilo-compucorp** [comment]
pchr-571-tablet
pchr-571-mobile

**Comment by AkA84** [comment]
Ok so the end result in mobile is different. Is it possible to avoid it? (so keepining "Request Toil" and "Request Leave" on the same line with "Use TOIL" in a new line)

**Comment by danilo-compucorp** [comment]
Hmm not sure if I agree / understand. 
If the "Request TOIL" is supposed to be "apart" from the other 2 buttons, why do we need to keep it side-by-side with "Request Leave" and "Use TOIL" in a new line ?

**Comment by AkA84** [comment]
Totally fair point, I agree. Please point it out though in your ticket when you send it to testing

PR approved

**Pull request merged**